pallets/foreign-asset-creator/src/lib.rs

```diff
@@ -152,6 +176,8 @@ pub mod pallet {
 			AssetIdToForeignAsset::<T>::insert(&asset_id, &foreign_asset);
 			ForeignAssetToAssetId::<T>::insert(&foreign_asset, &asset_id);
 
+			T::OnForeignAssetCreated::on_asset_created(&foreign_asset, &asset_id, &min_balance);
```

Comment:
I dont see a reason why we would need the min_balance here, I guess we can leave it out for now

Reply:
My idea was to make it general purpose. If particular kind of hook does not require `min_balance`, it can just ignore, I guess.

Reply:
I think we can make that more generic in the future if we would like, but I guess it's ok if we leave it as well